badgefactor.php

```diff
@@ -1265,7 +1265,10 @@ public function get_user_achievements ( $author_id ) {
         $achievements = array();
 
 	    //Re-fetch our data if the transient has expired.
-	    if ( false === ( $badges = get_transient( 'custom_badgeos_user_achievements' ) ) ) {
+	    if ( false 
```

Comment:
Ca semble être un erreur : 
`Fatal error: Call to undefined function false() in /app/plugins/badge-factor/badgefactor.php on line 1269`